src/JobCloud.BE.Shared/Helpers/SalaryRangeParser.cs

```diff
@@ -0,0 +1,37 @@
+﻿using JobCloud.BE.Domain.Models;
+
+namespace JobCloud.BE.Shared.Helpers
+{
+    public static class SalaryRangeParser
+    {
+        public static SalaryRange Parse(string value)
+        {
+            if (!value.Contains("-"))
+            {
+                throw new ArgumentException("Invalid input format. Expected format: 'number-number'.");
```

Comment:
Idea: How about `Invalid input for SalaryRange...`? Would be more descriptive

Reply:
Thats throw only for logging because it will be triggered only by scrappers. There will be the logging handler that will display context so message would be like "[JobCloud.BE.Shared] [SalaryRangeParser.Parse([ugly c# context values])] Invalid input format. Expected format: 'number-number'."